src/index.html

```diff
@@ -36,7 +36,7 @@
               </svg>
             </a>
             <!-- discord -->
-            <a title="Discord" href="https://discord.gg/CacsuuzsBq" class="flex h-6 w-6 justify-center text-gray-300"
+            <a title="Discord" href="https://discord.gg/altstore-delta-625714187078860810" class="flex h-6 w-6 justify-center text-gray-300"
```

Comment:
I'm imagining after the rename, we'll have a new discord.gg URL too though?

Reply:
If that's the case I can create a new invite and then fix it

Reply:
But I see no reason why this invite shouldn't keep working

Reply:
Why not use bitly or cloudflare url redirect so we can change the invite code without updating the page

Reply:
Whatever you want me to do i was trying to get ahead of everything

Reply:
Ya know, we should share a bitly account for this (or whatever account that tracks stats) so we can keep tabs on the number of clicks from different sources.